get-started/part2.md

```diff
@@ -321,7 +321,7 @@ username, repository, and tag names so that the image uploads to your
 desired destination. The syntax of the command is:
 
 ```shell
-docker tag image username/repository:tag
+docker tag <image id> <username>/<repository id>:<tag name>
```

Comment:
I'm a bit unsure about these; `image id` may actually be confusing, because it's not just the `IMAGE ID` that can be used; it can be any reference to an existing image (which could be the image's ID, or `image:tag`, or `image@sha256:digest`). Same for `<repository id>` (as it's not an _id_).

I realise these things are difficult to get right; even more so because Docker Hub uses `username/repository`, but "username" is only by convention on Docker Hub to use "username" as the first part (it's not part of the distribution spec; e.g. `foo/bar/baz:tag` would be acceptable on some registries.

Perhaps using some more obvious names would help; e.g.

```
docker tag mylocalimage username/someimage:sometag
```

But not sure if that makes it more clear; also, some actual example is already shown below, so... still on the fence if the proposed changes make this more clear